src/pages/Dashboard/DashboardV2.js

```diff
@@ -960,8 +957,8 @@ export default function DashboardV2() {
             <div className="token-table-wrapper App-card">
               <div className="App-card-title">
                 <Trans>GLP Index Composition</Trans>{" "}
-                {chainId === AVALANCHE && <img src={avalanche16Icon} alt={t`Avalanche Icon`} />}
-                {chainId === ARBITRUM && <img src={arbitrum16Icon} alt={t`Arbitrum Icon`} />}
+                {chainId === AVALANCHE && <img src={avalanche16Icon} alt="Avalanche Icon" />}
```

Comment:
I've added CHAIN_ICONS to config/chains

it think it's better to use this, because now we have AVALANCHE_FUJI chain and it would be easier to manage chainId based icons for all chains in one place

Reply:
Awesome! I am doing a lot of icon changes on PR #368 so I will make this change on same PR.